### PR TITLE
Issue 1876 webapi

### PR DESCRIPTION
**Pull request by crgrieve**
Reviewing and updating WebAPI docs for v8 as per issue #1876

### CODE REVIEW

File: Reference/Routing/WebApi/authorization.md [hunk 1]
```diff
@@ -44,12 +43,9 @@ To secure your controller based on front-end membership use the attribute: `Umbr
 
 *It's important to note the namespace since we have another class called MemberAuthorizeAttribute in a different namespace that is used for MVC.*
 
-There are 4 parameters that can be supplied to control how the authorization works:
+There are 3 parameters that can be supplied to control how the authorization works:
 
 ```csharp
-// Flag for whether to allow all site visitors or just authenticated members
-// This is the same as applying the [AllowAnonymous] attribute
-bool AllowAll
```
Comment (jmayntzhusen) [inline Reference/Routing/WebApi/authorization.md]: Did the `AllowAll` parameter never exist or has it been removed in v8? 🤔 If it still exists in v7 we should probably have this article versioned 🙂

File: Reference/Routing/WebApi/routing.md [hunk 2]
```diff
@@ -19,6 +18,8 @@ All plugin based Umbraco api controllers will be routed under the url path of:
 
 ~/Umbraco/[YourAreaName]/[YourControllerName]
 
+* See [here](/index.md) for more information on implementing these controllers.
```
Comment (jmayntzhusen) [inline Reference/Routing/WebApi/routing.md]:
```suggestion
* See [here](index.md) for more information on implementing these controllers.
```

File: Reference/Routing/WebApi/authorization.md (hunk 1 above)
Comment (crgrieve) [reply Reference/Routing/WebApi/authorization.md]: I checked in the v7 branch of code and although it was available, it was obsolete in favour of [AllowAnonymous].

I am happy to create a versioned one if this is preferred approach :)

Comment (jmayntzhusen) [reply Reference/Routing/WebApi/authorization.md]: No that's perfectly fine then - thanks a lot! 🙂🎉

**Pull request merged**